internal/messaging/subscriber.go

```diff
@@ -37,6 +62,9 @@ func NewNatsSubscriber(ctx context.Context, nc *nats.Conn, durable string, handl
 		jetstream.ConsumerConfig{
 			FilterSubjects: subjects,
 			Durable:        durable,
+			AckWait:        1 * time.Hour,
```

Comment:
Just to clarify, could you let me know why we need this?

Reply:
I will add a comment to explaing what this is for